packages/dds/sequence/src/sharedString.ts

```diff
@@ -17,14 +17,14 @@ import {
 	refHasTileLabel,
 } from "@fluidframework/merge-tree/internal";
 
-import { SharedSegmentSequence } from "./sequence.js";
+import { SharedSegmentSequence, type ISharedSegmentSequence } from "./sequence.js";
 import { SharedStringFactory } from "./sequenceFactory.js";
 
 /**
  * Fluid object interface describing access methods on a SharedString
  * @alpha
  */
-export interface ISharedString extends SharedSegmentSequence<SharedStringSegment> {
```

Comment:
This is what's getting fixed in this change. Having an interface expend a concreate class leaks the class types into our interface types, defeating the separation the interfaces are supposed to provide.